timescaledb/how-to-guides/hyperfunctions/time-weighted-average.md

```diff
@@ -0,0 +1,35 @@
+# Time-weighted average
+Timescale's time weighted average is implemented as an aggregate that
+weights each value using last observation carried forward (LOCF), or linear
```

Comment:
For those things that have in-depth blog posts about them (specifically this and percentile approximation, I think it'd be good to link out to the blog post for more information?

Reply:
Sure, I'll add them in.